mkdocs.yml

```diff
@@ -0,0 +1,382 @@
+site_name: AllenNLP
```

Comment:
Is this file autogenerated at all? Can we make it autogenerated?  Like, have part of it checked in as a separate file, then the part that's autogenerated gets concatenated to the checked in part before docs are generated?

Reply:
Sure, I can remove the part of it that is autogenerated 👍